src/react/ErrorPage.jsx

```diff
@@ -28,12 +30,17 @@ class ErrorPage extends Component {
                 description="error message when an unexpected error occurs"
               />
             </p>
+            {message !== null ? (
+              <div role="alert">
+                <p>{message}</p>
+              </div>
+            ) : (<React.Fragment key="nothing"></React.Fragment>)}
           </div>
         </div>
         <div className="row">
           <div className="col">
             <Button
-              buttonType="primary"
+              buttonType="primary mr-2"
```

Comment:
Is this right margin intentional? If so, you might want to move it the `className` prop. The `buttonType` prop technically should only the Bootstrap button types, with additional class names coming from the `className` prop.

Reply:
meant to remove this.  It was some ui testing when I added another button. I will remove.